routes/admin.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+
+const router = express.Router()
+
```

Comment:
add a comment mentioning the basic functionality of route

Reply:
Added the comments for the routes